homeassistant/components/v2c/sensor.py

```diff
@@ -90,11 +89,12 @@ async def async_setup_entry(
     """Set up V2C sensor platform."""
     coordinator: V2CUpdateCoordinator = hass.data[DOMAIN][config_entry.entry_id]
 
-    entities: list[Entity] = [
-        V2CSensorBaseEntity(coordinator, description, config_entry.entry_id)
-        for description in TRYDAN_SENSORS
-    ]
-    async_add_entities(entities)
+    async_add_entities(
+        [
```

Comment:
We can remove the parenthesis. That will make it a generator expression instead of a list comprehension.